src/com/sap/piper/MapUtils.groovy

```diff
@@ -63,6 +63,19 @@ class MapUtils implements Serializable {
         m.putAll(updates)
     }
 
+    static private def getByPath(Map m, def key) {
+        if (m == null) return null
```

Comment:
What do you have in mind here so that `null` is an expected value? What about a `NullPointerException`. Is it an expected use-case to receive `null` here. If 'yes' that would be fine, but for me that looks more like an error. In case this is an expected use-case we might express this by a corresponding test.

Reply:
No, I think we can expect a Map all the time.